R/zzz.r

```diff
@@ -19,6 +19,10 @@
 .onLoad <- function(...) {
   backport_unit_methods()
 
+  .zeroGrob <<- grob(cl = "zeroGrob", name = "NULL")
```

Comment:
If I understand the documentation to `.onLoad()` correctly (https://stat.ethz.ch/R-manual/R-devel/library/base/html/ns-hooks.html), this should be `grid::grob`.

Reply:
I don’t think so - where do you see that? `.onLoad()` is called after the namespace is loaded

Reply:
I was commenting based on the following sentence. It's possible I misinterpreted it.

> Note that the code in `.onLoad` and `.onUnload` should not assume any package except the base package is on the search path. Objects in the current package will be visible (unless this is circumvented), but objects from other packages should be imported or the double colon operator should be used.

Reply:
I think this is a throwback to ancient times when packages could assume default packages were loaded (e.g. stats) and thus didn't need to be imported in the namespace file... grid is imported by ggplot2 so it's all good